build-system/tasks/check-links.js

```diff
@@ -133,7 +116,7 @@ function isLinkToFileIntroducedByPR(link) {
  * @param {string} markdown Original markdown.
  * @return {string} Markdown after filtering out whitelisted links.
```

Comment:
Update comment to match function name change.

Reply:
Good catch!